.github/scripts/release-docs.sh

```diff
@@ -53,6 +53,11 @@ VERSION_STRING_REPLACEMENT="version = \"${CHANNEL_VERSION}\""
 awk -v REPLACEMENT="${VERSION_STRING_REPLACEMENT}" '{gsub(/version = \"edge\"/, REPLACEMENT); print}' docs/config.toml > docs/config.toml.tmp
 mv docs/config.toml.tmp docs/config.toml
 
+# In docs/config.toml, change version to CHANNEL_VERSION instead of latest or older version
+DOC_VERSION_STRING_REPLACEMENT="version = \"${CHANNEL_VERSION}\""
+awk -v REPLACEMENT="${DOC_VERSION_STRING_REPLACEMENT}" '{gsub(/^[ \t]*version = "(latest|v.*)"/, REPLACEMENT); print}' docs/config.toml > docs/config.toml.tmp
```

Comment:
When testing this it looks like the indentation is off: 
<img width="1040" alt="image" src="https://github.com/radius-project/docs/assets/54363786/da02df03-5cad-441f-ab45-82012ec3a6b5">

Reply:
Yes but the indentation is a stylistic choice here it doesn't effect the outcome, if you run the `hugo server` command the indentation has no effect on the variable value as it's just a text file. And looking at alternatives I can add a indentation to the `awk` command but that will end up indentating the other `version` variable above. So it's just a stylistic choice.

Reply:
Please see the toml schema for more details: https://github.com/toml-lang/toml?tab=readme-ov-file#example

Reply:
Hey i fixed it by changing the command!

Here's an explanation of the new one:
In this sed command:

-i.bak edits the file in place and creates a backup with the .bak extension.
-E enables extended regular expressions, which allow for more complex patterns.
The pattern ^( *)[ \t]*version = \"(latest|v.*)\" is used to match lines with version = "latest" or version = "v..."
^( *)[ \t]* captures leading spaces (or tabs) before version = and stores them in \1.
\1 in the replacement part puts the captured indentation back, ensuring the replacement line has the same leading whitespace as the original.
${DOC_VERSION_STRING_REPLACEMENT} is inserted after the captured whitespace.